### ISSUE TITLE
v2.2 (I think) prevented any connection (wifi, LAN) thus preventing critical update from downloading

### ISSUE THREAD

**Issue by jchhenderson** [objective-see/LuLu#330]
Well, that was a scare; I have a 2016 MBP and the internet was telling me this problem is due to either a logic board fail or the Manu-bar/fingerprint scanner not working.

My Mac was working fine until I restarted it this morning.  upon restarting, my Mac Big Sur 11.2.1) would not let me log in without completing a critical update.  However it was also suddenly refusing to connect to LAN or WIFI in order to download said update.  So I was Kida stuck.

Safe mode worked (LAN, WIFI OK) as did recovery mode.  I tried reinstalling macOS and resetting N/PRAM etc to no avail.

Eventually, I twigged that LuLu might be the problem given that  v
![IMG_3012](https://user-images.githubusercontent.com/34644672/108874442-0fa6ed00-75f4-11eb-8e19-5b870e1273df.jpeg)
2.0.0 just killed any connection for me.

I removed LuLu in safe mode, restarted and the critical updated downloaded and completed just fine.

I am not sure if this is a problem with LuLu or a restriction in Big Sur.  Either way, if others have this problem, they will at least know why.

**Comment by quentinlesceller** [comment]
Follow the instructions here https://github.com/objective-see/LuLu/issues/284#issuecomment-786857975

**Comment by jchhenderson** [comment]
thanks, but to be clear, it worked normally and only didn't work during the critical update.  That advice sounds like it is for situations where it wont work at all (or have I misunderstood)?

**Comment by quentinlesceller** [comment]
Hi @jchhenderson, this is the exact same situation. MBP works normally and after a restart you are always stuck on this screen. I added my fix here so someone with the same issue can find the solution quickly online.

**Comment by jchhenderson** [comment]
Hi @quentinlesceller, thanks for you reply - I appreciate it! - but it doesn't seem the same situation to me; I am able to access WIFI quite OK after a normal restart.

It was only during this particular "critical update" that it wouldn't work as it won't get to the login screen (where WIFI works).

Also, I do not have `com.objective-see.lulu.extension.systemextension` in `Library/SystemExtensions/{UUID}`

**Comment by quentinlesceller** [comment]
Yes it was the same. Wifi works well even during the update but the updater cannot reach the update probably because LuLu is blocking the connection to the outside.

**Comment by jchhenderson** [comment]
OK, thanks.  How do I delete a file that isn't actually there?

**Comment by quentinlesceller** [comment]
I think removing the app did remove the System Extension. For some reasons, on my side, when installed and remove with brew that didn't do the trick.

**Comment by jchhenderson** [comment]
so, reinstalling will put that extension back?

**Comment by quentinlesceller** [comment]
Reinstalling and launching the app will most definitely put the extension back and this bug will very likely happen again until @objective-see fixes the underlying issue.

**Comment by jchhenderson** [comment]
OK, thanks for clarifying.
